src/utils.js

```diff
@@ -1,10 +1,18 @@
 import execa from 'execa';
 
-// TODO: this is not optimal naming since it may cause confusion with 'spawn()' from 'child_process' module;
-
 // use execa to spawn a better child process
-/* eslint-disable-next-line import/prefer-default-export */
-export const spawn = (cmd, opts = { stdio: 'inherit' }) => {
+export const spawnProcess = (cmd, opts = { stdio: 'inherit' }) => {
   const [file, ...args] = cmd.split(/\s+/);
   return execa(file, args, opts);
 };
+
+export const isDefined = (variable) => {
```

Comment:
There is also a `_.isUndefined` function that you could use from `lodash`.

Reply:
I wasn't aware that this function existed. I have replaced it.